gulpfile.babel.js

```diff
@@ -65,7 +65,14 @@ gulp.task('lint:json', () => {
 
 gulp.task('format:json', () => {
   return gulp
-    .src(['./data-source/json/*.json'])
+    .src(['./data/*/*.json'])
```

Comment:
Dont care about `/data` as its `.gitignore`

Reply:
It was failing format tests without this